docs/generated/changelog.html

```diff
@@ -28,6 +28,12 @@ <h2>Version 0.10.5</h2>
           versions to 0 for major version updates
         </li>
         <li>Removes jest-expect-message, which was making test error messages less useful</li>
+        <li>
+          HttpAgent now generates a nonce to ensure that calls are unique by default. If you want to
+          opt out or provide your own nonce logic, you can now pass an option of
+          <pre>disableNonce: true</pre>
+          during the agent initalization.
```

Comment:
Can we say something about removing existing calls to `  agent.addTransform(makeNonceTransform());` ? 

```suggestion
          during the agent initialization.
```